quiz/backend/helpers/tests/test_generate.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from helpers.gen_username.generate import generate_username
+
+
+def test_usernames_file_read(monkeypatch, nouns_temp_file, adjectives_temp_file):
```

Comment:
I would rename this test to `test_username_generator_with_no_args` and then I would add `test_username_generator_with_args`.